02-Context-or-Narrative/State-of-PureScript-Documentation-2019.md

```diff
@@ -0,0 +1,550 @@
+>
+> The contributor responds sarcastically, "Oh! Delegation, of course! I hadn't thought about that...."
+>
+> There's actually an assumption going on here: "If 'free' rice means you can take as much as you want, then 'free' labor implies you can take as much as you want." But that isn't how labor works. If you don't pay for labor, you get less.
+>
+> Even if everyone could help, there's still limitations due to coordinating efforts: who does what, when, and where, and all without affecting someone else's work.
+>
+> ~ summary of the 'Why Not Just... Pattern' in [The Hard Parts of Open Source](https://youtu.be/o_4EX4dPppA?t=257)
+
+One shouldn't trust everyone on the internet. (I know, big surprise.)
+
+For example, consider the `event-stream incident`. The maintainer unknowingly gave write-access to a malicious actor. Practically everyone depends on this code. The actor injected code that could be used to steal bitcoins:
+- [initial issue](https://github.com/dominictarr/event-stream/issues/116)
+- [injected code explanation](https://github.com/dominictarr/event-stream/issues/116#issuecomment-441759047)
+- [original maintainer's response](https://github.com/dominictarr/event-stream/issues/116)
+- [a sarcastic blog post summarizing this class of attack](https://hackernoon.com/im-harvesting-credit-card-numbers-and-passwords-from-your-site-here-s-how-9a8cb347c5b5)
+
+New maintainers, once found and properly screened, have to be supported (which takes time and thought), so that they can actually fulfill their role. If they decide to leave soon after starting, then one has wasted time and effort.
+
+(Related interpretation sections: [Knowing whom to trust with write access, defining best workflow procedures, and providing necessary support](https://github.com/chexxor/purescript-documentation-discussion/blob/master/01-Sources-and-Interpretation/All-Interpretations.md#knowing-whom-to-trust-with-write-access-defining-best-workflow-procedures-and-providing-necessary-support))
+
+#### Why Not Just Fund the Language Developers?
+
+Been there, done that. Someone took it down. ([PureScript Open Collective]((https://opencollective.com/purescript))
+
+Our viewpoint of this situation:
+- Fans of PureScript wanted to throw money at the project. Why? Perhaps because
+    - they want to help the project and its goals
+    - they dont have the time or mental capacity to contribute
+- Several people used OpenCollective to indicate they'd like to pledge money to the "github.com/purescript/purescript" project.
+- Several weeks later, someone claimed that project's corresponding OpenCollective page and turned off the public page.
+- We believe this was because there was no clear vision as to how the funds would be spent.
+
+Perhaps this idea could be revisited in the future. For now, we cannot say.
+
```

Comment:
So, I believe the commit that we thought had been removed was still included. The above rendering is definitely different than the original one I wrote.

Reply:
This is different than the one you wrote? I remember reviewing a version of the PR which added this. Are you wanting to revise this part? Just wondering about your intention behind this comment.

Reply:
See the commit here: https://github.com/chexxor/purescript-documentation-discussion/pull/50/commits/164b36e222ecda1608cf338d4a456b8f0bb21be3

Reply:
You added more context that made me think my original rendering in the above commit wasn't really accurate. Thus, I went with your rendering.
For context, I'm referring to [your comment here](https://github.com/chexxor/purescript-documentation-discussion/pull/53#discussion_r270639509)

Reply:
Oh right, yeah I remember seeing that you pretty much pasted my comment into here with that commit, but I interpreted that as your adopting my version of the story without needing any extra edits from yourself.

Reply:
Well, yeah! I thought you rendering was great and didn't need anything, haha!

Reply:
Anything else here? Or can we resolve this conversation?